### PR TITLE
Blog post for Hacktoberfest 2017

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Blog post to announce out participation in the Hacktoberfest 2017.

### CODE REVIEW

File: source/_posts/2017-09-29-hacktoberfest.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+date: 2017-09-29 08:00:00 +0000
+date_formatted: "September 29, 2017"
+author: FAbian Affolter
+author_twitter: fabaff
+comments: true
+categories: Community
+og_image: /images/blog/2017-09-hacktoberfest/hacktoberfest-2017.png
+---
+
+Home Assistant will join this year for [Hacktoberfest], an event organised by DigitalOcean and GitHub to support and celebrate open source. The idea is that open source projects like Home Assistant will gather a bunch of entry-level bugs, features, and documentation enhancements and that you, a current or future contributor, will fix them. If you submit four pull-requests during the month of October you will have earned yourself a limited edition Hacktoberfest T-shirt!
```
Comment (frenck) [inline source/_posts/2017-09-29-hacktoberfest.markdown]: **Spelling & Grammar**:
- `organised` -> `organized`
- The following can be removed: `the month of`
- Missing comma after `October` -> `during October, you will`

File: source/_posts/2017-09-29-hacktoberfest.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Participating in Hacktoberfest"
+description: "Home Assistant will be participating in Hacktoberfest to help people to get started with open-source."
+date: 2017-09-29 08:00:00 +0000
+date_formatted: "September 29, 2017"
+author: FAbian Affolter
```
Comment (balloob) [inline source/_posts/2017-09-29-hacktoberfest.markdown]: Fabian

File: source/_posts/2017-09-29-hacktoberfest.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ - [Sign up for Hacktoberfest][Hacktoberfest-reg]
+ - [List of entry-level issues for Home Assistant][issues]
+ - [Instructions to setup your development environment][dev-env]
+ - [Home Assistant Development Chat][dev-chat]
+ - [Home Assistant Development Forums][dev-forum]
+ - [Hacktoberfest website][Hacktoberfest]
+
+Are you not a programmer but still want to contribute to Home Assistant? Check out our [list of entry-level issues for the Home Assistant website][issues-doc].
+
+Our participation for the [Hacktoberfest 2016][hackt-2016] was a huge success. Joun us to repeat t
```
Comment (dale3h) [inline source/_posts/2017-09-29-hacktoberfest.markdown]: - `participation for the` -> `participation for`
- `Joun` -> `Join`
- Unfinished sentence

File: source/_posts/2017-09-29-hacktoberfest.markdown (hunk 1 above)
Comment (dale3h) [inline source/_posts/2017-09-29-hacktoberfest.markdown]: - `Home Assistant will join this year for [Hacktoberfest]` -> I am having trouble coming up with the correct grammar for this. Does anyone else have ideas?
- `will fix them` -> `will help us fix them`

Comment (frenck) [reply source/_posts/2017-09-29-hacktoberfest.markdown]: How about: `Home Assistant joins Hacktoberfest this year! Hacktoberfest is an event....`

Comment (happyleavesaoc) [reply source/_posts/2017-09-29-hacktoberfest.markdown]: `Home Assistant will participate in [Hacktoberfest]`?

**Comment by fabaff** [comment]
Thanks for your support.

**Pull request merged**